src/contrib/testkits/Akka.TestKit.Xunit2/Akka.TestKit.Xunit2.nuspec

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <metadata xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <id>@project@</id>
+    <title>@project@@title@</title>
+    <version>@build.number@</version>
+    <authors>@authors@</authors>
+    <owners>@authors@</owners>
+    <description>TestKit for writing tests for Akka.NET using xUnit 2.0.</description>
```

Comment:
Could you update `Akka.TestKit.Xunit2.nuspec` so it says 1.9.2 in the same way as here?